std/file.d

```diff
@@ -4474,6 +4479,7 @@ void rmdirRecurse(ref DirEntry de)
 //"rmdirRecurse(in char[] pathname)" implementation. That is needlessly
 //expensive.
 //A DirEntry is a bit big (72B), so keeping the "by ref" signature is desirable.
+@safe
```

Comment:
Can you put the function attribute inline with the function declaration, as you previously did?

Reply:
Sure

Reply:
Done